spec/features/create_etd_spec.rb

```diff
@@ -0,0 +1,235 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Create a new ETD', type: :feature do
+  let(:etd_base_url) { 'etd-stage.stanford.edu' }
+  # dissertation id must be unique; D followed by 9 digits, e.g. D123456789
+  let(:dissertation_id) { format('D%09d', Kernel.rand(1..999_999_999)) }
+  let(:dissertation_title) { 'Integration Testing of ETD Processing' }
+  let(:xml_from_registrar) do
+    # see https://github.com/sul-dlss/hydra_etd/wiki/Data-Creation-and-Interaction#creating-new-etd-records
+    <<-XML
+    <DISSERTATION>
+      <dissertationid>#{dissertation_id}</dissertationid>
+      <title>#{dissertation_title}</title>
+      <type>Dissertation</type>
+      <vpname>Patricia J. Gumport</vpname>
+      <readerapproval>Not Submitted</readerapproval>
+      <readercomment> </readercomment>
+      <readeractiondttm> </readeractiondttm>
+      <regapproval>Not Submitted</regapproval>
+      <regactiondttm> </regactiondttm>
+      <regcomment> </regcomment>
+      <documentaccess>Yes</documentaccess>
+      <schoolname>School of Medicine</schoolname>
+      <degreeconfyr>2020</degreeconfyr>
+      <reader>
+        <sunetid>dedwards</sunetid>
+        <name>Edwards, Doris</name>
+        <type>int</type>
+        <univid>05358772</univid>
+        <readerrole>Doct Dissert Advisor (AC)</readerrole>
+        <finalreader>Yes</finalreader>
+      </reader>
+      <reader type="int">
+        <univid>05221995</univid>
+        <sunetid>rkatila</sunetid>
+        <name>Katila, Riitta</name>
+        <readerrole>Doct Dissert Reader (AC)</readerrole>
+        <finalreader>No</finalreader>
+      </reader>
+      <univid>05543256</univid>
+      <sunetid>dkelley</sunetid>
+      <name>Kelley, DeForest</name>
+      <career code="MED">Medicine</career>
+      <program code="MED">Medical</program>
+      <plan code="ANT">Neurology</plan>
+      <degree>PHD</degree>
+    </DISSERTATION>
+    XML
+  end
+
+  # may need to increase this: file uploads and submission to registrar
+  # Capybara.default_max_wait_time = 5
```

Comment:
Can this be removed? These settings are in spec_helper

Reply:
My point is that these tests in particular may require a longer wait time;  I can change it to "see spec_helper to increase this" sort of thing - would that suffice?

Reply:
Sure but we've already increased it way beyond the default: https://github.com/sul-dlss/infrastructure-integration-test/blob/master/spec/spec_helper.rb#L32

Reply:
whoa!  30 seconds!  ok - removing.